ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -4,12 +4,12 @@ workflow:
     pre:
     - chain: ipi-openstack-pre
     test:
-      - ref: openshift-e2e-test
```

Comment:
Why is testing commented out?

Reply:
Testing. There is currently a strange problem which causes resources to be left around in vexxhost when the test fails. 
I have not been able to pinpoint the cause, so is best to turn the testing off while we get this patch merge, or we'll kill vexxhost and block other testing. 
see https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/15992/rehearse-15992-pull-ci-openshift-installer-master-e2e-openstack-ipi/1366613758591373312  that one failed and it left resources around. 
so lets be safe until we get everything lined up.